src/include/gcc.h

```diff
@@ -11,6 +11,7 @@
 
 /* GCC-specific attributes. */
 #define	WT_PACKED_STRUCT_BEGIN(name)					\
+	/* NOLINTNEXTLINE(misc-macro-parentheses) */	\
 	struct __attribute__ ((__packed__)) name {
```

Comment:
Clang Tidy is complaining that ``name`` isn't wrapped in brackets (usually to avoid operator precedence issues) even though that's not allowed in a struct definition.

I'm assuming that this is an unusual use of preprocessor macros that they haven't considered so I've suppressed the linter here.